workflows/cath-shared.nf

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env nextflow
+nextflow.enable.dsl=2
+
+params.chunk_size = 10
+
+params.cath_af_cli = 'cath-af-cli'
+params.cath_version = 'v4_3_0'
+
+params.publish_dir = "$workflow.launchDir/cath-test-hpc-${params.dataset_name}"
```

Comment:
This is probably workflow specific - maybe move this one back to the workflow?

Reply:
|Made that change and added results-*/ to gitignore